frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx

```diff
@@ -48,68 +48,65 @@ const getStoragesData = ({
     diskInterface: disk.getDiskInterface(),
     size: disk.getReadableSize(),
     storageClass: disk.getStorageClassName(),
+    metadata: { name: disk.getName(), type: disk.getType() },
   }));
 };
 
 export type VMDisksTableProps = {
   data?: any[];
   customData?: object;
-  row: RowFunction;
-  columnClasses: string[];
+  Row: RowFunction;
+  loaded: boolean;
 };
 
 const NoDataEmptyMsg = () => <EmptyBox label="Disks" />;
 
-export const VMDisksTable: React.FC<VMDisksTableProps> = ({
-  data,
-  customData,
-  row: Row,
-  columnClasses,
-}) => {
+const HeaderFacroty = (columnClasses: string[]) => () =>
```

Comment:
please don't name it Factory. Can this be something simpler like `getHeader` ?

Reply:
ok, it's `getHeader` now :-)